protocol/messenger_communities.go

```diff
@@ -1161,12 +1162,16 @@ func (m *Messenger) RequestToJoinCommunity(request *requests.RequestToJoinCommun
 		privilegedMembers := community.GetFilteredPrivilegedMembers(map[string]struct{}{})
 
 		for _, member := range privilegedMembers[protobuf.CommunityMember_ROLE_OWNER] {
+			//TODO(frank): need support resending following message? if yes then we need to ensure SendMessageToControlNode
+			// will invoke SendPrivate internally
```

Comment:
If there are privileged members in the community, `!community.PublicKey().Equal(community.ControlNode()` should always be true. In this case, `SendMessageToControlNode` will use `SendPrivate`.

That's because only holder of owner-token can grant privileges (mint tokens). Once owner-token is minted, control node is no longer the community id, but the user who minted the token.